tests/ci/run_posix_tests.sh

```diff
@@ -40,3 +40,12 @@ if [[ "${AWSLC_FUZZ}" == "1" ]]; then
   echo "Testing building fuzz tests."
   run_build -DFUZZ=1
 fi
+
+# Lightly verify that uncommon build options does not break the build. Fist
+# define a list of typical build options to verify the special build option with
+build_options_to_test=("" "-DBUILD_SHARED_LIBS=1" "-DCMAKE_BUILD_TYPE=Release" "-DBUILD_SHARED_LIBS=1 -DCMAKE_BUILD_TYPE=Release" "-DBUILD_SHARED_LIBS=1 -DCMAKE_BUILD_TYPE=Release")
```

Comment:
Are the last two `build_options` duplicate elements?

Reply:
Ooops, I only removed the fips option not the entire test case.